app/tiebout.hoon

```diff
@@ -0,0 +1,253 @@
+::
+::::  /app/tiebout/hoon
+::
+/?  309
```

Comment:
Woah, I haven't seen one of these used in forever. I don't actually know how this operates across breaches that pass by this version etc. (Not that I've ever seen zuse version be changed in the first place.)

Reply:
removing